notary-server/Dockerfile

```diff
@@ -1,26 +1,6 @@
-FROM golang:1.19-alpine3.16 AS build
-
-RUN apk add --no-cache git make
-
-ENV NOTARYPKG github.com/theupdateframework/notary
-ENV TAG v0.7.0
-
-ENV GOFLAGS -mod=vendor
-
-WORKDIR /go/src/$NOTARYPKG
-RUN set -eux; \
-	git clone -b "$TAG" --depth 1 "https://$NOTARYPKG.git" .; \
-# https://github.com/notaryproject/notary/pull/1635
-	git fetch --depth 2 origin efc35b02698644af16f6049c7b585697352451b8; \
-	git -c user.name=foo -c user.email=foo@example.com cherry-pick -x efc35b02698644af16f6049c7b585697352451b8; \
-# https://github.com/notaryproject/notary/issues/1602 (rough cherry-pick of ca095023296d2d710ad9c6dec019397d46bf8576)
-	go get github.com/dvsekhvalnov/jose2go@v0.0.0-20200901110807-248326c1351b; \
-	go mod vendor; \
-# TODO remove for the next release of Notary (which should include efc35b02698644af16f6049c7b585697352451b8 & ca095023296d2d710ad9c6dec019397d46bf8576)
-	make SKIPENVCHECK=1 PREFIX=. ./bin/static/notary-server ./bin/static/notary-signer; \
-	cp -vL ./bin/static/notary-server ./bin/static/notary-signer /; \
-	/notary-server --version; \
-	/notary-signer --version
+FROM notary:base as base
+
+ARG BUILDKIT_SBOM_SCAN_STAGE=true
 
```

Comment:
🤔 I don't think keeping `base` as a separate stage is useful. This only adds the builder SBOM to the final image. I'd rather have users that want the other SBOM to just download it from the builder image rather than duplicating it into the other images.

So, we can drop these `FROM` and `ARG` lines and then update the `COPY` line:
```dockerfile
COPY --from=notary:0.7.0-base /notary-server ./
```

---

Perhaps `builder` is a more descriptive tag name since it isn't used as the base (starting point) of the other images?